integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/context/CcInstance.java

```diff
@@ -96,6 +102,7 @@ public static CcInstance getInstance() throws CCTestException {
         List<String> startupAPIProjectFiles = new ArrayList<>();
         boolean withCustomJwtTransformer = false;
         boolean withAnalyticsMetricImpl = false;
+        boolean isInterceptorCertInRouterTruststore = false;
```

Comment:
This variable name is misleading when its used. lets rename to `isInterceptorCertRequired` or something similar

Reply:
Renamed to isInterceptorCertRequired with https://github.com/wso2/product-microgateway/pull/2611/commits/6ff8e7d55d85dc362095542cce2f14efbdcfd023